internal/activity.go

```diff
@@ -269,6 +270,13 @@ func WithActivityTask(
 	} else {
 		deadline = startToCloseDeadline
 	}
+
+	logger.With(
+		zapcore.Field{Key: tagWorkflowType, Type: zapcore.StringType, String: *task.WorkflowType.Name},
+		zapcore.Field{Key: tagWorkflowID, Type: zapcore.StringType, String: *task.WorkflowExecution.WorkflowId},
+		zapcore.Field{Key: tagRunID, Type: zapcore.StringType, String: *task.WorkflowExecution.RunId},
+	)
```

Comment:
Does it already include activity type?

Reply:
Good point, I will add that as well

Reply:
Do we also want to add activity ID ?

Reply:
I added activity ID and activity type to it.

Reply:
Is ActivityID ok? As the number of them is potentially unbounded.